File: Example/CoreData/SyncKit/QSEmployee.m
//
//  QSEmployee.m
//  SyncKit
//
//  Created by Manuel Entrena on 30/12/2016.
//  Copyright © 2016 Manuel. All rights reserved.
//

#import "QSEmployee.h"

@implementation QSEmployee

+ (NSString *)primaryKey
{
    return @"identifier";
}

+ (NSString *)parentKey
{
    return @"company";
}

@end

File: Example/CoreData/SyncKit/QSEmployee.swift
//
//  QSEmployee.swift
//  SyncKit
//
//  Created by Jérôme Haegeli on 25.07.18.
//  Copyright © 2018 Manuel. All rights reserved.
//

import CoreData
import Foundation
import SyncKit

@objc(QSEmployee)
public class QSEmployee: NSManagedObject, QSPrimaryKey, QSParentKey {
    public class func primaryKey() -> String {
        return "identifier"
    }

    public class func parentKey() -> String {
        return "company"
    }
}
